### PR TITLE
Add tag prefix `v` in release link

### PR DESCRIPTION
**Pull request by dstotijn**

### CODE REVIEW

**Review by GeorgeMac — CHANGES_REQUESTED** [review]
Hey @dstotijn Thanks for the proposal here!

We've had a long tail of these `v` issues since we did some work in our gorelease script to support the fact we have a kind of Go monorepo here. The way we fixed it has lead to gorelease stripping the `v` off the version tag when we release. This is another one of those spots.

The more complete fix here would be to update the following line shown here:
https://github.com/flipt-io/flipt/blob/182662afddee677b9d2e7ce57b86abf7a47081ce/.goreleaser.yml#L9

```
-      - -s -w -X main.version={{ .Version }} -X main.commit={{ .Commit }} -X main.date={{ .Date }} -X main.analyticsKey={{ .Env.ANALYTICS_KEY }}
+      - -s -w -X main.version=v{{ .Version }} -X main.commit={{ .Commit }} -X main.date={{ .Date }} -X main.analyticsKey={{ .Env.ANALYTICS_KEY }}
```

Would you mind changing it there instead in this PR?
Then I will hapily approve and merge. It should then be fixed in the next patch release.
We're putting a patch release out this week as we have some other fixes in the pipeline.

**Review by markphelps — APPROVED** [review]
Thanks @dstotijn !! Our release process changed somewhat recently which messed w the tag. Much appreciated!

Edit: what @GeorgeMac said

**Comment by dstotijn** [comment]
Hey gang,
Made the recommended change. Maybe you can double check if `githubReleaseChecker.check` will still work, but because it uses `semver.ParseTolerant`, it should be fine with the leading "v".

File: .goreleaser.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ monorepo:
 builds:
   - main: ./cmd/flipt/.
     ldflags:
-      - -s -w -X main.version={{ .Version }} -X main.commit={{ .Commit }} -X main.date={{ .Date }} -X main.analyticsKey={{ .Env.ANALYTICS_KEY }}
+      - -s -w -X main.version=v{{ .Version }} -X main.commit={{ .Commit }} -X main.date={{ .Date }} -X main.analyticsKey={{ .Env.ANALYTICS_KEY }}
```
Comment (dstotijn) [inline .goreleaser.yml:9]: Alternatively, `.PrefixedTag` ([ref](https://goreleaser.com/customization/templates/)) _might_ work as well, but I haven't used that field before, so not sure.

E.g.
```suggestion
      - -s -w -X main.version={{ .PrefixedTag }} -X main.commit={{ .Commit }} -X main.date={{ .Date }} -X main.analyticsKey={{ .Env.ANALYTICS_KEY }}
```

File: ui/src/components/Footer.tsx [hunk 2]
```diff
@@ -13,7 +13,7 @@ export default function Footer() {
 
   const ref = () => {
     if (info?.isRelease && info?.version) {
-      return `v${info.version}`;
+      return info.version;
```
Comment (dstotijn) [inline ui/src/components/Footer.tsx:16]: `info.version` should come (indirectly) from `metadata.Server.GetInfo`, so should now have the "v" prefix already, passed down as-is from `main`.

File: .goreleaser.yml (hunk 1 above)
Comment (GeorgeMac) [reply .goreleaser.yml:9]: I think you might be right with that suggestion there.
Lets give `.PrefixedTag` ago @dstotijn 👍

**Comment by markphelps** [comment]
@all-contributors please add @dstotijn for code

**Comment by GeorgeMac** [comment]
Nice one @dstotijn ! 🙇

**Review by GeorgeMac — APPROVED** [review]

**Pull request merged**